napari/components/_window/__init__.py

```diff
@@ -2,10 +2,14 @@
 Custom Qt widgets that serve as native objects that the public-facing elements
 wrap.
 """
-from PyQt5.QtWidgets import QApplication as QtApplication
-
+from qtpy.QtWidgets import QApplication as QtApplication
+from os import environ
 from .view import Window
 
 from vispy import app
-app.use_app('pyqt5')
+
+if 'QT_API' in environ:
+    app.use_app(environ['QT_API'])
+else:
+    app.use_app('pyqt5')
```

Comment:
What's going on here? Could you add a comment clarifying?

Reply:
hmm yeah, it's maybe not ideal. If you look at the [qtpy](https://github.com/spyder-ide/qtpy) readme you'll see that you can specify which backend to use with the `environ['QT_API']` if multiple backends are present. If multiple are present then it will default to `PyQt5`, if just one is present then it will use that one. The problem is that choice needs to be communicated to vispy. This code will currently cause a conflict if just `PySide2` is present but the environ variable is not set. I'm not quite sure what to do - any advice?

Reply:
We could detect it and insert it into os.environ? Is there a way to inspect the backend that qtpy is using?

Reply:
Looks like `qtpy3API` might contain the currently-used API?

Reply:
qtpy.API, sorry

Reply:
Also .API_NAME

Reply:
Great find, thanks! Fixed!